app/controllers/api_controller/arbitration_profiles.rb

```diff
@@ -0,0 +1,43 @@
+class ApiController
+  module ArbitrationProfiles
+    def create_resource_arbitration_profiles(_type, _id, data)
+      validate_profile_data(data)
+      parse_set_provider(data)
```

Comment:
I just followed the conventions found [here](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api_controller/groups.rb#L7) and [here](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api_controller/parser.rb#L161)

suggestions for simplifying?